PhotoSketch/ColoringBook/sketch.js

```diff
@@ -0,0 +1,157 @@
+
+// Copyright (C) 2018 Runway AI Examples
+// 
+// This file is part of Runway AI Examples.
+// 
+// Runway-Examples is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// Runway-Examples is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with Runway.  If not, see <http://www.gnu.org/licenses/>.
+// 
+// ===========================================================================
+
+// RUNWAY
+// www.runwayapp.ai
+
+// PhotoSketch ColoringBook Demo:
+// Gets an image from Runway via HTTP
+// Made by jpyepez
+
+
+let p5canvas;
+let routeInput,
+    getButton, 
+    saveButton, 
+    colPicker, 
+    strokeSlider, 
+    clearButton;
+
+////////////////////
+// Runway Functions
+////////////////////
+
+// Get data, transform to image and draw to canvas
+
+let runwayGet = 'http://localhost:8000/data';
```

Comment:
Perhaps we can add a status icon or text that helps identify if the app is connected to Runway? Something like, ` Not Connected` and `Connected`. What do you think?

Reply:
Awesome! Added a label that pings the server and shows: 
* `Connected` - Runway is running, valid route 
* `Not Connected` - Runway not running or the route is not valid 

Tried to keep things simple and in P5 to avoid using listeners and other stuff that might not be familiar for users who haven't used vanilla JS or ES6+.

In some cases the status will display `Connected` if Runway is using a slightly different route, maybe from a previous workspace (i.e. `8000` instead of `8001`), but it should show the alert when trying to get the image, which asks you to check (1) the route and (2) that the model is running. This pretty much covers all possible flows while keeping the code readable and without getting too fancy.

Let me know if there's anything else. :)

Reply:
Looks great! thanks for this!